bucket/gradle.json

```
@@ -9,6 +9,12 @@
     "url": "https://services.gradle.org/distributions/gradle-7.5.1-all.zip",
     "hash": "db9c8211ed63f61f60292c69e80d89196f9eb36665e369e7f00ac4cc841c2219",
     "extract_dir": "gradle-7.5.1",
+    "env_set": {
+        "GRADLE_USER_HOME": "$dir\\.gradle"
+    },
+    "persist": [
+        ".gradle"
+    ],
```

```suggestion
    "persist": ".gradle",
```
Same simplification as above